pkg/drivers/kic/oci/cli_runner.go

```diff
@@ -88,15 +88,16 @@ func runCmd(cmd *exec.Cmd, warnSlow ...bool) (*RunResult, error) {
 	}
 
 	killTime := 19 * time.Second // this will be applied only if warnSlow is true
-	warnTime := 2 * time.Second
-	ctx, cancel := context.WithTimeout(context.Background(), killTime)
-	defer cancel()
+	warnTime := 10 * time.Second
```

Comment:
we could have two warnTimes based on the command, I would leave the default warnTime for other commands as 2 seconds, as if some commands take more than 2 seconds user needs to know there is something wrong with their docker

Reply:
I realized we don't actually need to change the warnTime -- the issue isn't that warnTime was too low, it was that we were printing out "please restart docker" on any error, instead of only on a DeadineExceeded error

I reverted this back to how it was